.github/workflows/publish-release.yml

```diff
@@ -31,6 +31,12 @@ jobs:
           git checkout -b release-${VERSION}
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew
+      - name: Update installer version
+        env:
+          GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
+        run: |
+          ./gradlew build updateInstallerVersion
+          git commit -m "update auto generated installer version"
```

Comment:
This has failed 
https://github.com/ballerina-platform/ballerina-distribution/runs/3055835469?check_suite_focus=true

Reason could be before commit you need to add it to index with `git add gradle.properties`